taxonomy/utils.py

```diff
@@ -104,6 +122,41 @@ def process_skills_data(course, course_skills, should_commit_to_db):
     return failures
 
 
+def process_program_skills_data(program, program_skills, should_commit_to_db):
+    """
+    Process program skills data returned by the EMSI service and update databased.
+
+    Arguments:
+        program (dict): Dictionary containing program data whose skills are being processed.
+        program_skills (dict): Program skills data returned by the EMSI API.
+        should_commit_to_db (bool): Boolean indicating whether data should be committed to database.
+    """
+    failures = []
+    for record in program_skills['data']:
+        try:
+            confidence = float(record['confidence'])
+            skill = record['skill']
+            skill_external_id = skill['id']
+            skill_data = {
+                'name': skill['name'],
+                'info_url': skill['infoUrl'],
+                'type_id': skill['type']['id'],
+                'type_name': skill['type']['name'],
+                'description': skill['description']
+            }
```

Comment:
nit: this can be converted into a util transform method. The same code is being used in process_course_skills_data.